packages/grant-explorer/src/features/round/ViewProjectDetails.tsx

```diff
@@ -117,6 +139,62 @@ function AboutProject(props: { projectToRender: Project }) {
   const userGithub = projectToRender.projectMetadata.userGithub;
   const projectGithub = projectToRender.projectMetadata.projectGithub;
 
+  useEffect(() => {
+    if (projectToRender?.projectMetadata.owners) {
+      const credentials: ProjectCredentials =
+        projectToRender?.projectMetadata.credentials ?? {};
+
+      if (!credentials) {
+        return;
+      }
+      const verify = async () => {
+        const newVerifiedProviders: { [key: string]: VerifiedCredentialState } =
+          { ...verifiedProviders };
+        for (const provider of Object.keys(verifiedProviders)) {
+          const verifiableCredential = credentials[provider];
+          if (verifiableCredential) {
+            newVerifiedProviders[provider] = await isVerified(
+              verifiableCredential,
+              verifier,
+              provider,
+              projectToRender
+            );
+          }
+        }
+
+        setVerifiedProviders(newVerifiedProviders);
+      };
+      verify();
+    }
+  }, [projectToRender?.projectMetadata.owners]); // eslint-disable-line react-hooks/exhaustive-deps
+
+  const getVerifiableCredentialVerificationResultView = (provider: string) => {
+    switch (verifiedProviders[provider]) {
+      case VerifiedCredentialState.VALID:
+        return (
+          <span className="rounded-full bg-teal-100 px-3 inline-flex flex-row justify-center items-center">
+            <ShieldCheckIcon
+              className="w-5 h-5 text-teal-500 mr-2"
+              data-testid={`${provider}-verifiable-credential`}
+            />
+            <p className="text-teal-500 font-medium text-xs">Verified</p>
+          </span>
+        );
+      case VerifiedCredentialState.INVALID:
+        return (
+          <span className="rounded-full bg-red-100 px-3 inline-flex flex-row justify-center items-center">
+            <XCircleIcon
+              className="w-5 h-5 text-white mr-2"
+              data-testid={`${provider}-verifiable-credential-unverified`}
+            />
+            <p className="text-white font-medium text-xs">Invalid</p>
+          </span>
+        );
```

Comment:
@nfrgosselin i included the state for unverified credentials to match application detail page in round manager. it's not explicitly in the figma design

should we keep this?